### DISCUSSION TITLE
Mix of the cp and mt contigs on the assembly graph

### DISCUSSION THREAD

**Discussion by DenisGoryunov** [Kinggerm/GetOrganelle#204] [Q&A] [answered]
Hi,
I have a number of questions after cp genome assembly. Could you help me please?

1. There is a mix of the cp and mt contigs on the largest graph. I was trying to assemble a cp genome while. Why i can see both cp and mt contigs on the single graph? What is wrong with my settings?
2. Is the contig coverage depth sufficient for the cp genome assembly?

3. Which settings i have to optimise for the complete genome assembly?

Many thanks,
Denis

[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/9747119/get_org.log.txt)
![graph-2](https://user-images.githubusercontent.com/10725817/194899416-c246622d-fe8a-4935-bede-229e3e681903.png)

**Comment by Kinggerm** [comment, marked as answer]
It is normal to see this mixture. Please read the GetOrganelle paper for more.

For improving the assembly, try the options following here: https://github.com/Kinggerm/GetOrganelle/wiki/FAQ#what-should-i-do-with-incomplete-resultbroken-assembly-graph

**Comment by DenisGoryunov** [comment]
Thanks for your response!
